### DISCUSSION TITLE
Addon Controls - Editing objects

### DISCUSSION THREAD

**Discussion by appsparkler** [storybookjs/storybook#14234] [Ideas]
Hello Super Folks!

Storybook is awesome!

I'm currently going through the tutorial and my understanding of CDD has 🚀 since I started.  Thank you for everything :)

I was wondering if it would be possible to format the output of `array/object` in the controls `textarea`:

![image](https://user-images.githubusercontent.com/12084821/111090412-a57bcb00-8555-11eb-9e38-5b2a7b672bed.png)

Good Day :)

**Comment by shilman** [comment]
Storybook 6.2 has a nice JSON editor for this: https://github.com/storybookjs/storybook/pull/12824

To upgrade: 

```
npx sb@next upgrade --prerelease
```
